### ISSUE TITLE
Odd Rack Elevation Size Rendering

### ISSUE THREAD

**Issue by ebboyer** [netbox-community/netbox#14479]
### Deployment Type

Self-hosted

### NetBox Version

v3.6.4

### Python Version

3.10

### Steps to Reproduce

1. Navigate to /dcim/racks/ and click on any rack. If this issue is observed, you'll see each U rendered at 45 pixels high, as seen in this screenshot:

![Screenshot 2023-12-04 at 5 00 34 PM](https://github.com/netbox-community/netbox/assets/129799095/5c285bda-dc7c-4ef0-b87b-53883b31ae10)

Perhaps this is an issue with our specific instance / plugins. It is present on both our staging server and our production server that are running v.3.6.4, which we recently upgraded from v3.5.3. I looked in /netbox/netbox/config/parameters.py and verified that the default rendering should be 220 pixels wide by 22 pixels tall. 

![Screenshot 2023-12-04 at 5 07 12 PM](https://github.com/netbox-community/netbox/assets/129799095/af1ff074-89e6-4d76-8353-0a56b9f6b24b)

Here are the plugins we're running
netbox-floorplan                    0.1.1
netbox-inventory                   1.6.0
netbox-topology-views         3.8.1

### Expected Behavior

I would expect each U in a rack elevation to be rendered to match the default pixel height specified in /netbox/netbox/config/parameters.py 

### Observed Behavior

Each rack U is rendered to be 45 pixels high, even for empty rack Us, which should default to 22 pixels high.

**Comment by jeremystretch** [comment]
> I looked in /netbox/netbox/config/parameters.py and verified that the default rendering should be 220 pixels wide by 22 pixels tall.

Right, those are the default values. What values are actually defined in your current configuration?

**Comment by ebboyer** [comment]
Thanks Jeremy! That's the strange thing. As far as I can tell, we aren't explicitly setting the value anywhere else in any files, so I would think that the defaults should be what we're using. Other than in that parameters file, where else might the values be getting set?

**Comment by jeremystretch** [comment]
You can navigate to "Admin > Current Config" in the UI to see the current values, if any (assuming you have superuser permission).
![rack_elevations](https://github.com/netbox-community/netbox/assets/13487278/21981bc7-a7b3-4f9d-9072-42d9578c3580)

**Comment by ebboyer** [comment]
Thanks for the tip! When I navigated to that page, interestingly, it didn't seem to populate either of those fields.

![Screenshot 2023-12-12 at 9 07 20 AM](https://github.com/netbox-community/netbox/assets/129799095/7109967d-206f-46a2-8a90-1aade520bae9)

However, I used the web UI to make a new config and that worked. When I did that, it populated with 45 as being set as the current value. So, I changed that to the default of 22 and voila!

![Screenshot 2023-12-12 at 11 50 07 AM](https://github.com/netbox-community/netbox/assets/129799095/93356c55-de2f-416d-9aef-3f75ecfcda28)

Thanks again for the help!
